web/appsettings.json

```
@@ -2,6 +2,8 @@
   "ApplicationInsights": {
     "ConnectionString": ""
   },
+  "BlobConnectionString" : "DefaultEndpointsProtocol=https;AccountName=bnhpropertyimages;AccountKey=6kjyiLjh7FBBr/G+vCf9mWiGDdbsAuHjCXfj7mrsl06ai5H7uRMhPMHlxqx6JxknsBfqY8pMyKI4+AStAOwVHQ==;EndpointSuffix=core.windows.net",

```

leave as empty placeholder

```suggestion
  "BlobConnectionString" : "",
```